CONTRIBUTING.md

```diff
@@ -12,16 +12,18 @@ you will need to change this.
 
 Once you've gotten MySQL running you can run Lighthouse. It will take care of the rest.
 
-## Connecting
+## Connecting (PS3)
 
-PS3 is difficult to set up, so I will be going over how to set up RPCS3 instead. A guide will be coming for PS3 closer
-to release. You can also follow this guide if you want to learn how to modify your EBOOT.
+You can use [UnionPatcher's Remote Patch utility](https://github.com/LBPUnion/UnionPatcher) to manually patch your EBOOT.BIN; it works over the network and automatically downloads, decrypts, patches, encrypts, and uploads your PSN and Disc EBOOTs.
```

Comment:
```suggestion
You can use the [UnionPatcher Remote Patch utility](https://github.com/LBPUnion/UnionPatcher) to manually patch your EBOOT.BIN; it works over the network and automatically downloads, decrypts, patches, encrypts, and uploads your PSN and Disc EBOOTs.
```
I think this works better grammatically as "UnionPatcher" doesn't own the patcher utility

Reply:
How about "the Remote Patch utility in UnionPatcher"?

Reply:
that works too

Reply:
done